application/tests/build.xml

```diff
@@ -0,0 +1,30 @@
+<project name="testreport" default="phpunit" basedir=".">
```

Comment:
@biechao The project name is not good. Consider, please.

Reply:
The project name has been changed.